j2ee.sun.appsrv/licenseinfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/j2ee/sun/ide/sunresources/wizards/ResourceWizard.template</file>
+        <file>src/org/netbeans/modules/j2ee/sun/sunresources/beans/wizard.dtd</file>
```

Comment:
I suppose that this is used here:
https://github.com/apache/incubator-netbeans/blob/91d94516c76ab50e8b420f70633694c025ca516e/j2ee.sun.appsrv/src/org/netbeans/modules/j2ee/sun/sunresources/beans/Wizard.java#L98-L115

Reply:
This is the same situation as #722 the DTD can carry the comment and should.